Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -906,46 +920,15 @@ private async Task<List<DeleteError>> removeObjectsAsync(string bucketName, List
         /// <returns></returns>
         public async Task<IObservable<DeleteError>> RemoveObjectAsync(string bucketName, IEnumerable<string> objectNames, CancellationToken cancellationToken = default(CancellationToken))
         {
-            if (objectNames == null)
```

Comment:
what will happen if `objectNames == null` ? Shouldn't this code be not removed?

Reply:
This function calls the new implementation.